tdrs-frontend/src/selectors/auth.js

```diff
@@ -62,4 +62,6 @@ export const accountCanViewAdmin = (state) =>
 
 export const accountCanViewKibana = (state) =>
   accountStatusIsApproved(state) &&
+  (selectUser(state)?.email?.includes('@acf.hhs.gov') ||
```

Comment:
not sure its worth noting as an inline comment here that the developer role + acf email yields the kibana tabs but endpoint is still inaccessible. this might only be relevant to dev team and qasp reviewers since developer role doesnt exist in prod. @elipe17

Reply:
![2805p3](https://github.com/raft-tech/TANF-app/assets/63075587/0d8ea1b6-c3da-4e78-b2c7-ee601d6f5b5d)

Reply:
Good catch, I will remove developer!